src/align/align.jl

```diff
@@ -0,0 +1,24 @@
+module Align
+
+using Compat
+using Docile
+using Docile: @doc, @doc_str
```

Comment:
We don't need to use Docile here if we use Julia's docstrings. Docile functionality don't depend on this `using` statements (we can use Docile even on modules that weren't meant to use it).